variables.tf

```diff
@@ -495,6 +489,18 @@ variable "extra_script" {
 ##############
 ###Alarms
 ##############
+variable "enable_asg_events_notify" {
+  description = "Whether to create AutoScaling Group with Notification support, via SNS Topics"
+  type        = bool
+  default     = false
+}
+
+variable "create_asg_sns_topic" {
```

Comment:
Do we need this variable or we can use the above to imply the above?

Reply:
The variable allows using an external sns topic that is not created using this module